python/cudf/cudf/core/column_accessor.py

```diff
@@ -350,7 +349,7 @@ def _pad_key(self, key: Any, pad_value="") -> Any:
         return key + (pad_value,) * (self.nlevels - len(key))
 
     def replace_level_values(
-        self, mapper: Union[Mapping[Any, Any], Callable], level: Optional[int]
+        self, mapper: Union[Mapping[Any, Any], Callable], level: int = None
```

Comment:
```suggestion
        self, mapper: Union[Mapping[Any, Any], Callable], level: int
```

Let's make this a required (non-optional arg)